lib/importexport/helpers.js

```diff
@@ -102,6 +102,7 @@ helpers.importFile = function(params, cb){
 };
 
 var _importZip = function(filePath, cb, retries){
+    return cb('_importZip calledddddddddddddddddddddddddddddddddddddddddddddddddddd');
     retries = retries || 0;
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Remove this code after the "return" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AUnreachableCode)